content/_releases/22/16.md

```diff
@@ -18,11 +18,11 @@ If you are one of power users of our platform you might have noticed a pop-up wa
 with a message **Not enough quota to start task** once or twice. In reality this
```

Comment:
If you are one our many power users you may have noticed from time to time a pop-up

Reply:
corrected as you've suggested